### PR TITLE
IGAPP-524: Add jpal to settings screen

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by maxammann — APPROVED** [review]

File: native/src/routes/settings/createSettingsSections.js [hunk 1]
```diff
@@ -113,7 +125,21 @@ const createSettingsSections = ({ setSetting, t, languageCode, cityCode }: Creat
...
-      }
+      },
+      ...(!buildConfig().featureFlags.jpalEvaluation
+        ? []
+        : [
+            {
+              accessibilityRole: 'none',
+              title: `JPAL ${t('evaluation')}`,
+              description: t('evaluationDescription'),
+              getSettingValue: (settings: SettingsType) => settings.trackingEnabled,
```
Comment (maxammann) [inline native/src/routes/settings/createSettingsSections.js:136]: I think this settings should be named `settings.jpalEnabled` as future users of the app will not necessarily want it enabled.

**Review by maxammann — APPROVED** [review]

File: native/src/routes/settings/createSettingsSections.js [hunk 2]
```diff
@@ -113,7 +125,21 @@ const createSettingsSections = ({ setSetting, t, languageCode, cityCode }: Creat
             throw Error('This error was thrown for testing purposes. Please ignore this error.')
           }
         }
-      }
+      },
+      ...(!buildConfig().featureFlags.jpalEvaluation
```
Comment (maxammann) [inline native/src/routes/settings/createSettingsSections.js:129]:
```suggestion
      ...(!buildConfig().featureFlags.jpalTracking
```

File: translations/translations.json [hunk 3]
```diff
@@ -2714,7 +2714,11 @@
       "about": "Über {{appName}}",
       "privacyPolicy": "Datenschutzerklärung",
       "version": "Version: {{version}}",
-      "openSourceLicenses": "Open-Source-Lizenzen"
+      "openSourceLicenses": "Open-Source-Lizenzen",
+      "evaluation": "Evaluation",
```
Comment (maxammann) [inline translations/translations.json:2718]:
```suggestion
      "tracking": "Evaluation",
```

File: translations/translations.json [hunk 4]
```diff
@@ -2714,7 +2714,11 @@
...
+      "evaluationDescription": "Durch das Senden von Tracking-Daten Integreat langfristig verbessern.",
```
Comment (maxammann) [inline translations/translations.json:2719]:
```suggestion
      "trackingDescription": "Durch das Senden von Evaluations-Daten Integreat langfristig verbessern.",
```

File: native/src/routes/settings/createSettingsSections.js (hunk 1 above)
Comment (klinzo) [reply native/src/routes/settings/createSettingsSections.js:136]: Sorry @SteffiStuffel :D @maxammann is probably right. We should call it `jpalTrackingEnabled` though imo.

File: native/src/routes/settings/createSettingsSections.js [hunk 5]
```diff
@@ -113,7 +125,21 @@ const createSettingsSections = ({ setSetting, t, languageCode, cityCode }: Creat
...
+        ? []
+        : [
+            {
+              accessibilityRole: 'none',
+              title: `JPAL ${t('evaluation')}`,
```
Comment (klinzo) [inline native/src/routes/settings/createSettingsSections.js:134]: I'd say this should be translated as a whole. I also think it is J-PAL.

**Comment by klinzo** [comment]
Tested on Android :+1:

**Pull request merged**